.github/pull_request_template.md

```diff
@@ -1,27 +1,21 @@
 #### Pull Request Checklist
 
+- [ ] All changes are Go version 1.15 compliant
+- [ ] Your PR passes all CI checks
 - [ ] If this is your first time contributing, we recommend you read the [Code
   Contribution Guidelines](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md)
-- [ ] All changes are Go version 1.12 compliant
-- [ ] The code being submitted is commented according to [Code Documentation and Commenting](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md#CodeDocumentation)
+   - [ ] The code being submitted is commented according to [Code Documentation and
+  Commenting](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md#CodeDocumentation)
+   - [ ] Commits have a logical structure according to [Ideal Git Commit
+  Structure](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md#IdealGitCommitStructure)
 - [ ] For new code: Code is accompanied by tests which exercise both
   the positive and negative (error paths) conditions (if applicable)
-- [ ] For bug fixes: Code is accompanied by new tests which trigger
+- [ ] For bug fixes: If possible code is accompanied by new tests which trigger
   the bug being fixed to prevent regressions
-- [ ] Any new logging statements use an appropriate subsystem and
-  logging level
-- [ ] Code has been formatted with `go fmt`
-- [ ] Protobuf files (`lnrpc/**/*.proto`) have been formatted with
-  `make rpc-format` and compiled with `make rpc`
-- [ ] New configuration flags have been added to `sample-lnd.conf`
+- [ ] Any new logging statements use an appropriate subsystem and logging level
 - [ ] For code and documentation: lines are wrapped at 80 characters
   (the tab character should be counted as 8 characters, not 4, as some IDEs do
   per default)
-- [ ] Running `make check` does not fail any tests
-- [ ] Running `go vet` does not report any issues
-- [ ] Running `make lint` does not report any **new** issues that did not
-  already exist
 - [ ] All commits build properly and pass tests. Only in exceptional
   cases it can be justifiable to violate this condition. In that case, the
   reason should be stated in the commit message.
```

Comment:
Could this be collapsed into "Your PR passes all CI checks"? Now that we have the commit check, we don't really make this exception anymore?

Reply:
Yes, good idea. What do you think about the current phrasing?

Reply:
Yeah that looks good!